src/utils/signature.ts

```diff
@@ -0,0 +1,4 @@
+import { SignJWT } from "@pagopa/io-react-native-jwt";
+
+export const getSignedJwt = async (unsignedJwt: string, signature: string) =>
+  await SignJWT.appendSignature(unsignedJwt, signature);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** I'm honestly curious what's the value of this module, being it's just plain wrapper‏

Reply:
this wrapper avoids using `@pagopa/io-react-native-jwt` directly and installing it as a dependency. If you think it is superfluous we can eliminate it, np.

Reply:
Ah, I get it. I have no strong opinion about it. Would it be a problem for the app to **directly** reference the package? (it will be referenced anyway as dependency)